xrdsst/controllers/service.py

```diff
@@ -260,6 +293,52 @@ def disable_descriptions(self):
                                           self.app.pargs.description,
                                           self.app.pargs.notice)
 
+    @ex(help="List service access", arguments=[(['--client'], {'help': 'Client id', 'dest': 'client'}),
+                                               (['--description'], {'help': 'Service description id', 'dest': 'description'})])
+    def list_access(self):
+        active_config = self.load_config()
+
+        missing_parameters = []
+        if self.app.pargs.client is None:
+            missing_parameters.append('client')
+        if self.app.pargs.description is None:
+            missing_parameters.append('description')
+        if len(missing_parameters) > 0:
+            BaseController.log_info('The following parameters missing for listing access rights for services: %s' % missing_parameters)
+            return
+
+        self.list_access_rights(active_config, self.app.pargs.client, self.app.pargs.description)
+
+    @ex(help="Delete service access", arguments=[(['--ss'], {'help': 'Security server name', 'dest': 'ss'}),
+                                                 (['--client'], {'help': 'Client id', 'dest': 'client'}),
+                                                 (['--description'], {'help': 'Service description id', 'dest': 'description'}),
+                                                 (['--service'], {'help': 'Service id', 'dest': 'service'}),
+                                                 (['--sclient'], {'help': 'Service client id', 'dest': 'sclient'})])
+    def delete_access(self):
+        active_config = self.load_config()
+
+        missing_parameters = []
+        if self.app.pargs.ss is None:
+            missing_parameters.append('ss')
+        if self.app.pargs.client is None:
+            missing_parameters.append('client')
+        if self.app.pargs.description is None:
+            missing_parameters.append('description')
+        if self.app.pargs.service is None:
+            missing_parameters.append('service')
+        if self.app.pargs.sclient is None:
+            missing_parameters.append('sclient')
+        if len(missing_parameters) > 0:
+            BaseController.log_info('The following parameters missing for deleting access rights for services: %s' % missing_parameters)
+            return
+
+        self.delete_access_rights(active_config,
```

Comment:
Also i see in your commands, that normally parse the parameters in the "remote" method, but I think it's better if we parse the parameters here so we don't need to go deeper in the code to check if the parameter could be multiple or if we must send a list or a string separated by ','

Reply:
Updated